ARTICLE-1.markdown

```diff
@@ -104,3 +105,22 @@ No Member-Owner may transfer her or his Membership or any right
 arising there from.  Any attempted assignment or transfer shall
 be wholly void and shall confer no rights on the intended
 assignee or transferee.
+
+### Section 1.5  Maintaining Membership
+
+In order to maintain Membership status, Member-Owners must
+work a minimum of 800 hours per year with the Cooperative. Work
+outside of the Cooperative is permitted, so long as the minimum
+hours to the Cooperative are met.
+
```

Comment:
800 hours are counted at the end of each financial year
If a person does not complete 800 hours, board takes a decision
If a person becomes a member in mid term, then pro rated hours will be considered for criteria